tests/sub_group_mask/sub_group_mask_extract_bits.cpp

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests to check sub_group_mask extract_bits()
+//
+*******************************************************************************/
+
+#include "sub_group_mask_common.h"
+
+#define TEST_NAME sub_group_mask_extract_bits
+
+namespace TEST_NAMESPACE {
+
+using namespace sycl_cts;
+#ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
+
+template <typename T>
+void get_expected_bits(T &out, uint32_t mask_size, int pos) {
+  if (pos >= mask_size - 1) return;
+  int init;
+  if (pos % 2 == 0)
+    init = 1;  // 01
+  else
+    init = 2;  // 10
```

Comment:
```suggestion
    init = 0b10;
```

Reply:
this commend was applied [here](https://github.com/KhronosGroup/SYCL-CTS/pull/202/commits/7b8b881e2665a5081119f502def79cecae04f2c3#diff-cfbc9b5df3268d3ff2e7c4f881ebbc0094b882799d25411270d0c817fc3fdd3aR25)